common/models/hook.js

```diff
@@ -22,6 +22,13 @@ module.exports = function (Hook) {
     })
   }
 
+  Hook.getPlaylist = function (playlistID, cb) {
+    console.log("Trying to get playlist ", playlistID)
+    spotify.getPlaylist(playlistID).then(results => {
+      cb(null, results)
+    })['catch'](function(error){console.log('caught', error)});
```

Comment:
Why `['catch']` here, rather than .catch? Just curious, think I might learn something :)

Reply:
https://doxdox.org/jmperez/spotify-web-api-js#src-spotify-web-api.js-constr.prototype.setpromiseimplementation
https://github.com/promises-aplus/promises-spec/blob/master/implementations.md

Looks like the Promise rabbit hole goes even deeper than I thought. The real answer is that that's what I found when I googled. I should probably learn my A+ promise core API. We should switch to 2-arg `then` or `.catch`.

The nonchalant "Oh yeah you can definitely just set a property called `'catch'` and that's the same thing" reminded me of this talk:
https://www.youtube.com/watch?v=Wiw3YcwGwrU